### ISSUE TITLE
algolia search box troubles?

### ISSUE THREAD

**Issue by lukasmrtvy** [lycheeverse/lychee#895]
Hello,
Can We do something with results from pages containing algolia search box ? Thanks
```
➜  /tmp ./lychee https://www.algolia.com/search/
⠐ 31/31 [00:00:10] █████████████████████████ ✗ [400] https://res.cloudinary.com/hilnmyskv/image/upload/v1580296397/Algolia_com_Website_assets/logo-algolia-xenon-blue-full.png&quot | FailedIssues found in 1 input. Find details below.

[https://www.algolia.com/search/]:
✗ [404] https://1qdawl72tq-dsn.algolia.net/ | Failed: Network error: Not Found
✗ [400] https://cdn.cookielaw.org/ | Failed: Network error: Bad Request
✗ [404] https://www.facebook.com/algolia/&quot | Failed: Network error: Not Found
✗ [404] https://www.instagram.com/algoliahq/&quot | Failed: Network error: Not Found
✗ [400] https://res.cloudinary.com/hilnmyskv/image/upload/v1580296397/Algolia_com_Website_assets/logo-algolia-xenon-blue-full.png&quot | Failed: Network error: Bad Request
✗ [404] https://cdn.segment.com/analytics.js/v1/ | Failed: Network error: Not Found
✗ [404] https://www.algolia.com/contact/enterprise/&quot | Failed: Network error: Not Found
✗ [404] https://www.algolia.com/demorequest/&quot | Failed: Network error: Not Found
✗ [404] https://www.github.com/Algolia/&quot | Failed: Network error: Not Found
✗ [404] https://res.cloudinary.com/ | Failed: Network error: Not Found
✗ [404] https://www.algolia.com/for-open-source/&quot | Failed: Network error: Not Found
✗ [404] https://www.twitter.com/Algolia/&quot | Failed: Network error: Not Found
✗ [400] https://res.cloudinary.com/hilnmyskv/image/upload/v1527077656/Algolia_OG_image_m3xgjb.png&quot | Failed: Network error: Bad Request
✗ [404] https://www.algolia.com/&quot | Failed: Network error: Not Found

🔍 31 Total ✅ 16 OK 🚫 14 Errors (HTTP:14) 💤 1 Excluded
```

lychee v0.10.3

**Comment by mre** [comment]
The URLs in question are encoded in a `script` tag along with some other data:

```html
<script type="application/ld+json">
[
  {
    &quot;@context&quot;: &quot;http://schema.org&quot;,
    &quot;@type&quot;: &quot;Organization&quot;,
    &quot;name&quot;: &quot;Algolia&quot;,
    &quot;url&quot;: &quot;https://www.algolia.com/&quot;,
    &quot;logo&quot;: &quot;https://res.cloudinary.com/hilnmyskv/image/upload/v1580296397/Algolia_com_Website_assets/logo-algolia-xenon-blue-full.png&quot;,
    &quot;image&quot;: &quot;https://res.cloudinary.com/hilnmyskv/image/upload/v1527077656/Algolia_OG_image_m3xgjb.png&quot;,
    &quot;description&quot;: &quot;Fast, reliable and modern Search &amp; Discovery&quot;,
    &quot;address&quot;: {
      &quot;@type&quot;: &quot;PostalAddress&quot;,
      &quot;streetAddress&quot;: &quot;301 Howard St, 3rd floor&quot;,
      &quot;addressLocality&quot;: &quot;San Francisco&quot;,
      &quot;addressRegion&quot;: &quot;California&quot;,
      &quot;postalCode&quot;: &quot;94105&quot;,
      &quot;addressCountry&quot;: &quot;USA&quot;
    },
    &quot;sameAs&quot;: [
      &quot;https://www.linkedin.com/company/algolia/&quot;,
      &quot;https://www.facebook.com/algolia/&quot;,
      &quot;https://www.github.com/Algolia/&quot;,
      &quot;https://www.youtube.com/user/Algolia/&quot;,
      &quot;https://www.twitter.com/Algolia/&quot;,
      &quot;https://www.instagram.com/algoliahq/&quot;
    ],
   ...
</script>
```

I thought that's fixed in `master` by https://github.com/lycheeverse/lychee/pull/848, which excludes `<script>` tags by default. Seems like it doesn't cover your case.
Just tested it and when I put the raw HTML into a file it works as expected. Only when I use the URL which serves that content it doesn't work. Not sure why yet.

**Comment by mre** [comment]
My check for remote URLs was incorrect. I guess I fixed it in #899 and added some more tests to avoid future breakage.
@lukasmrtvy fyi

**Comment by mre** [comment]
This is fixed in `master` and will be released with the next version. Thanks for the report @lukasmrtvy.
